app/src/test/java/org/mozilla/fenix/RunBlockingCounterTest.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class RunBlockingCounterTest {
+
+    @Test
+    fun `GIVEN we call runBlockingCounter THEN the count should increase`() {
+        assertEquals(0L, RunBlockingCounter.runBlockingCount)
+        runBlockingIncrement {}
```

Comment:
Should we also test that the correct value is returned from runBlockingIncrement?

Note: I don't think tests are guaranteed to run in a specific order and static members don't get reset between tests so you'll need to zero the counter between tests, probably.

Reply:
> Note: I don't think tests are guaranteed to run in a specific order and static members don't get reset between tests so you'll need to zero the counter between tests, probably.

I think this is why the test failed on CI